cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -191,7 +240,7 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
           resultTable()
         }
 
-        p("Like other `SHOW` commands, the output can also be processed using `YIELD` / `WHERE` / `RETURN`.")
+        p("Of course, the output can also be processed using `YIELD` / `WHERE` / `RETURN`.")
```

Comment:
I understand that it is a bit tricky to vary the language in these sentences, but I'm not so fond of using `of course` since it might make a user feel stupid if they are not grasping it. Perhaps something that the docs team have more input on